src/aat/resources/features/F-108/F-103_Test_Data_Base.td.json

```
@@ -0,0 +1,22 @@
+{
+  "_guid_": "F-103_Test_Data_Base",
+  "title": "Elastic Search Internal Endpoint",
+  "productName": "CCD Data Store Elastic Search API",
+  "operationName": "internal search query",
```

```suggestion
  "operationName": "external search query",
```
(And updated where this is referred to in the feature file)